force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +123,24 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+        Integer v = 0;
+        Set<String> vowelList = new Set<String>();
+        vowelList.add('a');
+        vowelList.add('e');
+        vowelList.add('i');
+        vowelList.add('o');
+        vowelList.add('u');
+        System.debug(vowelList);
+        String newstr = str.toLowerCase();
+        for(Integer i = 0; i < newstr.length(); i++){
+            String c = newstr.substring(i, i + 1); // get individual characters from the string - hint use substring or split
+            System.debug(c);
+            if(vowelList.contains(c)){ // check if the character is a vowel
+                v++;  // if it is a vowel, increment the result Integer
```

Comment:
I would probably at least call this `count` or `vowelCount` or something rather than v.